modules/aws-eks-managed-node-groups/managed-launch-templates.tf

```diff
@@ -34,9 +34,9 @@ resource "aws_launch_template" "managed_node_groups" {
   }
 
   metadata_options {
-    http_endpoint               = try(var.context.http_endpoint, "enabled")
-    http_tokens                 = try(var.context.http_tokens, "required") #tfsec:ignore:aws-autoscaling-enforce-http-token-imds
-    http_put_response_hop_limit = try(var.context.http_put_response_hop_limit, 2)
+    http_endpoint               = try(var.managed_ng.http_endpoint, "enabled")
+    http_tokens                 = try(var.managed_ng.http_tokens, "required") #tfsec:ignore:aws-autoscaling-enforce-http-token-imds
+    http_put_response_hop_limit = try(var.managed_ng.http_put_response_hop_limit, 2)
```

Comment:
This will still work but for consistency, let' use the merged config local variable

```hcl
http_endpoint               = try(local.managed_node_group["http_endpoint"], "enabled")
http_tokens                 = try(local.managed_node_group["http_tokens"], "required") #tfsec:ignore:aws-autoscaling-enforce-http-token-imds
http_put_response_hop_limit = try(local.managed_node_group["http_put_response_hop_limit"], 2)
```

Reply:
Isn't this redundant? I just removed the redundancy while keeping the default settings

Reply:
I meant either we have to use `try` for all the defaults or use the existing approach with **merged map** variable(`local.managed_node_group`) but not both.